src/main/java/fi/okm/jod/yksilo/controller/profiili/KoulutusController.java

```diff
@@ -46,12 +47,12 @@ List<KoulutusDto> getAll(@AuthenticationPrincipal JodUser user) {
 
   @PostMapping
   @ResponseStatus(HttpStatus.CREATED)
```

Comment:
Technical specification delta: return value 201 instead of 200

Reply:
won't fix, spec will be changed